lib/Chunk.js

```diff
@@ -388,7 +542,10 @@ class Chunk {
 		};
 	}
 
-	hasModuleInGraph(filterFn, filterChunkFn) {
+	hasModuleInGraph(
+		/** @type {(m: Module) => boolean} */ filterFn,
```

Comment:
Would it be more consistent for these to be `@param` annotations on the method, instead of `@type` on the param?